docs/source/usrman/mpi4py.futures.rst

```diff
@@ -421,18 +421,39 @@ scanline is dumped to disk.
                for line in image:
                    f.write(line)
 
-The recommended way to execute the script is using the :program:`mpiexec`
+The recommended way to execute the script is by using the :program:`mpiexec`
 command specifying one MPI process and (optional but recommended) the desired
-MPI universe size [#]_. ::
+`MPI universe size <https://www.mcs.anl.gov/research/projects/mpi/mpi-standard/mpi-report-2.0/node111.htm>`_,
+which is provided either by a batch system or is set by the user via
+command-line arguments or environment variables. Below we provide
+examples for MPICH and Open MPI as well as `mpi4py`-specific way [#]_.
+In all of these examples, the :program:`mpiexec` command launches a single MPI
+process (the master) running the Python interpreter and executing the main
+script. When required, :mod:`mpi4py.futures` spawns 16 additional MPI processes
+(the children) to dynamically allocate the pool of workers. The master submits
```

Comment:
If you used master before, then here use workers, not children.

Reply:
How about this?
When required, :mod:`mpi4py.futures` spawns the pool of 16 worker MPI processes

Reply:
I liked the clarification between parenthesis, but it should have said `(the workers)`. I would keep the parenthesis, to match the previous style.  _A foolish consistency is the hobgoblin of little minds ..._